src/components/RbzTable.vue

```diff
@@ -369,6 +385,10 @@ export default defineComponent({
   padding: 0.5em;
 }
 
+.menu {
+  float: right;
+}
+
```

Comment:
Let's avoid using float, this should work:

```suggestion
.rbz-table .menu {
    display: inline-flex;
    justify-content: flex-end;
    width: 100%;
}
```